drivers/hmis/app/models/hmis/hud/assessment.rb

```diff
@@ -20,7 +20,6 @@ class Hmis::Hud::Assessment < Hmis::Hud::Base
     date_updated: 'Last Updated: Most Recent First',
   }.freeze
 
-  belongs_to :enrollment, **hmis_enrollment_relation, optional: true
```

Comment:
moved to ClientProjectEnrollmentRelated because order matters for `has_many :through`